_test/test_sqw_pixels/test_PixelDataBase.m

```diff
@@ -48,9 +49,14 @@
 
     methods
 
-        function obj = test_PixelData(~)
+        function obj = test_PixelDataBase(~)
             obj = obj@TestCase('test_PixelData');
             obj.warning_cache = warning('off','HORACE:old_file_format');
+            hc = hor_config;
+            if hc.saveable
+                obj.initial_mem_chunk_size = hc.mem_chunk_size;
+            end
+            hc.saveable = false;
```

Comment:
this is new funtion and it does not look like it would work as clean up object will be destroyed after exiting the the constructor function. We want to return previous value on distruction